prow/cmd/tide/main.go

```diff
@@ -106,7 +110,13 @@ func main() {
 	}
 	defer gc.Clean()
 
+	sig := make(chan os.Signal, 1)
+	signal.Notify(sig, os.Interrupt, syscall.SIGTERM)
```

Comment:
nit: why are we making the signal channel here instead of at the usage below?

Reply:
No reason, moved.